packages/react-charting/src/components/CommonComponents/CartesianChart.base.tsx

```diff
@@ -463,7 +465,14 @@ export class CartesianChartBase extends React.Component<IModifiedCartesianChartP
         ref={(rootElem: HTMLDivElement) => (this.chartContainer = rootElem)}
         onMouseLeave={this._onChartLeave}
       >
-        <FocusZone direction={focusDirection} className={this._classNames.chartWrapper} {...svgFocusZoneProps}>
+        {!this._isFirstRender && <div id={this.idForDefaultTabbableElement} />}
+        <FocusZone
+          direction={focusDirection}
+          className={this._classNames.chartWrapper}
+          defaultTabbableElement={`#${this.idForDefaultTabbableElement}`}
+          {...svgFocusZoneProps}
+        >
+          {this._isFirstRender && <div id={this.idForDefaultTabbableElement} />}
```

Comment:
We need this only when optimizeData flag is set to true

Reply:
Without flag ,it is fixing this for Bar charts.